database/config/config.js

```diff
@@ -0,0 +1,18 @@
+const {DB, DB_PASSWORD, DB_HOST, DB_PORT, DB_USER } = process.env;
+
+const config = {
```

Comment:
use connection pooling too, read about it.

Reply:
Done and read about it👍